src/NBitcoin/Policy/StandardTransactionPolicy.cs

```diff
@@ -81,7 +85,7 @@ public TransactionPolicyError[] Check(Transaction transaction, ICoin[] spentCoin
                     if(ScriptVerify != null)
                     {
                         ScriptError error;
-                        if(!VerifyScript(input, coin.TxOut.ScriptPubKey, coin.TxOut.Value, ScriptVerify.Value, out error))
+                        if(!this.VerifyScript(input, coin.TxOut.ScriptPubKey, coin.TxOut.Value, ScriptVerify.Value, out error))
```

Comment:
When I change like this I almost want to report all code style issues. LOL

Reply:
yeah I could not resist this one, but hold yourself.